app/build.gradle.kts

```diff
@@ -4,10 +4,9 @@ import config.ApplicationConfig.androidMinSdk
 import config.ApplicationConfig.androidTargetSdk
 
 plugins {
-    id(config.Plugins.appPlugin)
-    kotlin(config.Plugins.androidPlugin)
-    kotlin(config.Plugins.kapt)
-    id(config.Plugins.hiltPlugin)
+    alias(libs.plugins.appPlugin)
+    alias(libs.plugins.androidPlugin)
+    alias(libs.plugins.kaptPlugin)
 }
 
 android {
```

Comment:
@DeKaN do you think I should also move android versions from `buildSrc/config/ApplicationConfig` to `libs.versions.toml`? 🤔

Reply:
Gradle Version Catalog can be used only for dependencies, not for custom constants